tests/Aspire.Components.Common.Tests/RequiresDockerAttribute.cs

```diff
@@ -1,12 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Xunit;
 using Xunit.Sdk;
 
-namespace Aspire.Workload.Tests;
+namespace Aspire.Components.Common.Tests;
 
-[TraitDiscoverer("Aspire.Workload.Tests.RequiresDockerDiscoverer", "Aspire.Workload.Tests")]
+// class level attribute
```

Comment:
What is this comment for?

Reply:
`RequiresDockerAttribute` - the name by itself seemed unclear on what it was meant for. Maybe I should make this applicable to methods also.

Reply:
It says it applies to Class and Method in the AttributeUsage.

Can we just use this method instead of `RequiresDockerFact` and `RequiresDockerTheory`? Would you just annotate the method as:
```C#
[Fact]
[RequiresDocker]
```

Reply:
Yeah, that would work.

Reply:
Don't need to fix it here, but that would be nice to consolidate on a single way to do this.